backend/main/community_test.go

```diff
@@ -52,6 +53,7 @@ func TestCreateCommunity(t *testing.T) {
 	communityStruct := otu.GenerateCommunityStruct("account")
 	communityPayload := otu.GenerateCommunityPayload("account", communityStruct)
 	response := otu.CreateCommunityAPI(communityPayload)
+	fmt.Printf("%+v\n", response.Body)
```

Comment:
Is this just a debug statement while you were testing or is this something we need in the test?

Reply:
actually we may still need this. When running tests Golang does print out all the error logging in the server. I was encountering a weird IPFS time out error. So I've submitted this PR now Jackson will test locally to see if he gets the IPFS error, (it seems to be intermittent). Will need this log here for now.